WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -236,3 +237,21 @@ $(function() {
 Math.trunc = Math.trunc || function(x) {
   return x < 0 ? Math.ceil(x) : Math.floor(x);
 };
+
+// Setting up bootstrap-table
+$(function() {
+  // Hide loading box
+  $('table[data-toggle="table"]').bootstrapTable('hideLoading');
+  // Triggered when a sort arrow is clicked but after a table is sorted
+  $('table').on('sort.bs.table', function(e, name, order) {
```

Comment:
nit: There's no need to add a listener on this table unless it's a bootstrap table, so change this to `table[data-toggle="table"]`?